source/documentation/tools/upgrading.md

```diff
@@ -0,0 +1,38 @@
+# Upgrading RStudio
+
+When moving between RStudio releases there are some differences that require either user intervention or caution.
+
+## Why should I upgrade?
+
+Upgrading to the latest version of RStudio has a number of benefits including newer versions of R, improved packages management options.
```

Comment:
Suggestion to reword slightly to?

_Upgrading to the latest version of RStudio has a number of benefits, including a new version of R and improved package management options._

Reply:
Yep, that reads much nicer, thanks!